content/BindingMethods/BindingMethods.q.md

```diff
@@ -461,30 +461,30 @@ In the statement `$pstmt->bindValue(1, 'John');` on line 8 of `student.php`, wha
 
 Which statements correctly describe `1` and `John` on line 8 of `student.php`?
 
-- It contains the key and array value of the parameter.
+- These are the arguments of the `bindValue()` method.
 
-- It contains the key and array element of the parameter.
+- They represent the key and array value of the parameter.
 
-- It represents the arguments of the `bindValue()` method.
+- They represent the key and array element of the parameter.
```

Comment:
Please check the given answer. I think this is not the correct answer to the question. Thanks.